tests/unit/SidebarOrderTest.js

```diff
@@ -431,7 +430,7 @@ describe('Sidebar', () => {
                         expect(screen.queryAllByTestId('Pin Icon')).toHaveLength(1);
                         expect(screen.queryAllByTestId('Pencil Icon')).toHaveLength(1);
                         expect(lodashGet(displayNames, [0, 'props', 'children'])).toBe('One, Two');
-                        expect(lodashGet(displayNames, [1, 'props', 'children'])).toBe('Email Two owes $100.00');
+                        expect(lodashGet(displayNames, [1, 'props', 'children'])).toBe('Email Two paid $100.00');
```

Comment:
Why does this change from `owes` to `paid` if all we're doing in this PR is determining whether or not the GBR should be displayed?

Reply:
I was thinking about the same thing, but tests were failing, because they were receiving this

Reply:
Probably more places where “ hasOutstandingIOU” was doing something
Cc: @puneetlath

Reply:
Yeah, I assumed tests failining meant something probably broke in the logic. 😅

Will wait for input from @puneetlath.

Reply:
@akinwale updated, changed to previous, hasOutstandingIOU was needed for a correct behaviour

Reply:
It looks like this change is no longer in the code? I didn't see it when reviewing.